src/js/components/PlacementConstraintsPartial.js

```diff
@@ -119,10 +124,12 @@ export default class PlacementConstraintsPartial extends Component {
                 );
               })}
             </Select>
-            <FieldHelp>
-              Specify where your app will run.
-            </FieldHelp>
-            <FieldError>{operatorError}</FieldError>
+
+            {operatorError && <FieldError>{operatorError}</FieldError>}
+            {!operatorError &&
```

Comment:
Do we really only want to display an error OR the help message? I'd say when an error occured the help is more needed than ever. 🤔

Reply:
Lily wrote 

>   In a case like that, you should remove the help text and only have the error text

in the [issue comments](https://jira.mesosphere.com/browse/DCOS-20094?focusedCommentId=133266&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-133266) that is the reason for this decision. I am not married to this decision, should we ping it back to Lily?